blog/2024-05-29-oras-1.2.0-and-oras-go-2.5.0/index.mdx

```diff
@@ -75,16 +75,16 @@ Keeping track of manifest content download and upload has never been more intuit
 - Introduce `--platform` to oras attach for better multi-arch attaching experience, which allows adding referrer artifact to a specific sub-platform
 - Introduce `oras resolve` to get the digest of an artifact
 
-In addition, the overall user experience and performance are also enhanced in this release：
+In addition, the overall user experience and performance are also enhanced in this release:
 
 - Reduce authentication request count for several ORAS commands and support blob mounting across repositories in the same registry for `oras copy`
 - Improve error message based on [ORAS CLI error handling guidance](https://github.com/oras-project/oras/blob/v1.2.0/docs/proposals/error-handling-guideline.md)
 
 There are a few bug fixes and a deprecated feature in this release. For a concrete changelog, please see [ORAS CLI v1.2.0 Release Notes](https://github.com/oras-project/oras/releases/tag/v1.2.0).
 
-### Use ORAS CLI in terminal and CI/CD pipelines
+### Use ORAS CLI in terminal, Docker container and CI/CD pipelines
 
-ORAS installation binary is available on Winget, Homebrew, Snap, Docker, and GitHub. It can be installed via one simple command. Please see the installation guide for your environment.
+ORAS installation binary is available on Winget, Homebrew, Snap, GitHub and also provides Docker container image. It can be installed via one simple command. Please see the installation guide for your environment.
```

Comment:
```suggestion
ORAS installation binary is available on Winget, Homebrew, Snap, GitHub and Docker container image. It can be installed via one simple command. Please see the installation guide for your environment.
```